input/elasticapm/internal/modeldecoder/v2/model.go

```diff
@@ -845,6 +845,8 @@ type spanContextHTTP struct {
 type spanContextHTTPRequest struct {
 	// ID holds the unique identifier for the http request.
 	ID nullable.String `json:"id"`
+	// The http request body as a string
+	Body nullable.String `json:"body"`
```

Comment:
For transactions we use `nullable.Interface` because we also support using a map directly for `multipart/form-data` content, I don't know if this was for legacy reasons. There are currently no plans for doing the same for http-client bodies, for those `multipart/form-data` will be a simple string atm.

But I guess it wouldn't hurt to be a but more lenient here in case we decide to change this in the future.

Reply:
Done in [2d29b67](https://github.com/elastic/apm-data/pull/366/commits/2d29b67c28abe801cafff942a17a9fcd5e94ea95)